lib/modules/manager/regex/readme.md

```diff
@@ -13,21 +13,25 @@ The first two required fields are `fileMatch` and `matchStrings`. `fileMatch` wo
 In order for Renovate to look up a dependency and decide about updates, it then needs the following information about each dependency:
 
 - The dependency's name
-- Which `datasource` to look up (e.g. npm, Docker, GitHub tags, etc)
-- Which version scheme to apply (defaults to `semver`, but also may be other values like `pep440`)
+- Which [`datasource`](https://docs.renovatebot.com/modules/datasource/#supported-datasources) to look up (e.g. [npm](https://docs.renovatebot.com/modules/datasource/#npm-datasource), [Docker](https://docs.renovatebot.com/modules/datasource/#docker-datasource), [GitHub tags](https://docs.renovatebot.com/modules/datasource/#github-tags-datasource))
+- Which [version scheme](https://docs.renovatebot.com/modules/versioning/) to apply (defaults to [`semver`](https://docs.renovatebot.com/modules/versioning/#semantic-versioning), but also may be [other values](https://docs.renovatebot.com/modules/versioning/#supported-versioning) like [`pep440`](https://docs.renovatebot.com/modules/versioning/#pep440-versioning))
+
+<!-- prettier-ignore -->
+!!! note
+The capture groups below are similar to [available fields](https://docs.renovatebot.com/templates/#other-available-fields), but these available fields are not available in templates below, because most of those are coming from managers, and `regex` is a manager.
```

Comment:
It still valid, as those fieds are not yet available, you can only use captured groups from regex match.